tests/acceptance/cli_test.py

```diff
@@ -2346,10 +2368,10 @@ def test_logs_tail(self):
         self.dispatch(['up'])
 
         result = self.dispatch(['logs', '--tail', '2'])
-        assert 'c\n' in result.stdout
-        assert 'd\n' in result.stdout
-        assert 'a\n' not in result.stdout
-        assert 'b\n' not in result.stdout
+        assert 'y\n' in result.stdout
```

Comment:
Don't you like `a b c d` anymore?

Reply:
Now that we have hex strings in the output, `a b c d` are not really safe characters anymore :sweat_smile:

Reply:
Well, that's a good point.